test/JsBuiltIns/rlexe.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<regress-exe>
+  <test>
+    <default>
+      <files>ChakraLib.js</files>
+      <compile-flags>-args summary -endargs -LdChakraLib</compile-flags>
```

Comment:
> -LdChakraLib [](start = 43, length = 13)

Is this totally separate from the builtins? Does the JsBuiltIn code work without -LdChakraLib? Should these flags be connected?